modules/heat_conduction/src/userobjects/GapFluxModelPressureDependentConduction.C

```diff
@@ -66,6 +66,6 @@ GapFluxModelPressureDependentConduction::computeFlux() const
   const ADReal h_sum = _primary_hardness[_qp] + _secondary_hardness[_qp];
   const ADReal h_harmonic = 2 * _primary_hardness[_qp] * _secondary_hardness[_qp] / h_sum;
 
-  return _scaling * k_harmonic * (_primary_T[_qp] - _secondary_T[_qp]) * _normal_pressure[0] /
+  return _scaling * k_harmonic * (_primary_T[_qp] - _secondary_T[_qp]) * _normal_pressure[_qp] /
```

Comment:
Did the evaluation of the LM at the corresponding integration point change the results? If so, still matching analytical solution reasonably?

Reply:
I think it'd be good to have tests that are influenced by the interpolation of the normal pressure given that the object computes heat transfer based on that value. Is the contact pressure completely constant in all the tests? I am guessing this can be a reason why indexing as `[_qp]` wouldn't make a difference.